### DISCUSSION TITLE
Episode 4: A talk with Eugene Yokota!

### DISCUSSION THREAD

**Discussion by ckipp01** [ckipp01/tooling-talks#4] [New Speaker Announcement!]
![ToolingTalkEugene NEW (twitter)](https://user-images.githubusercontent.com/13974112/127203823-a837ffbd-895b-40b0-a902-74ef9aea5984.jpg)

At the end of August I'll be sitting down with Eugene to talk about sbt, build tools, and all sorts of developer tooling. If you have any questions you'd like asked or topics covered for the episode, feel free to leave them here! No guarantee that they'll be asked, but they'll help me know what people want to hear!

### Find Eugene online

[@eed3si9n](https://twitter.com/eed3si9n) on Twitter

[@eed3si9n](https://github.com/eed3si9n) on GitHub

[https://eed3si9n.com/](https://eed3si9n.com/)

**Comment by jatcwang** [comment]
Thanks for all your work on SBT over the years. It has easily improved by 2~3x since I first started writing Scala. 

Q: If you can change anything in SBT (design or implementation) without any constraints, what would you change / do differently?

**Comment by ckipp01** [comment]
Dropping some of the questions that I got on Twitter for this just so they can all be kept in the same place:

- when’s sbt 2 coming out, what Scala version will it run on (2.13, or 3?), and what can we expect to contain?
- tell us about at least one of your lesser-known open-source projects that you wish were better known?
- to whatever extent you‘re able to talk about it, tell us what you’re working on at Twitter and what it’s like working on tooling there? any surprises, any learnings?
- Also favorite place to grab lunch post-pandemic.
- reflections on your years at Typesafe/Lightbend? what do you make of how the company has (and hasn’t) changed over the years?
- Why Scala? Of all the stuff out there… why Scala?

**Comment by Iltotore** [comment]
I would know how do you compare SBT to Mill. Li Haoyi has [his opinion about SBT](https://www.lihaoyi.com/post/SowhatswrongwithSBT.html) but I would know your opinion about SBT and Mill.

**Comment by TonioGela** [comment]
Thanks for all your work on SBT Eugene. Can I ask you what will be the future of giter8? Will it be included in sbt 2? Will it be rewritten from scratch? Will it be dropped at all? Will it be ported to scala 3?

**Comment by kevin-lee** [comment]
Hey Eugene, these are my questions. 🙂
* Could you tell me when I can start to write sbt plugin using Scala 3?
* I feel like it's not that common to work on build tools. How did you start working on sbt?
* I can see that you contribute to so many open-source projects. How do you manage them all?
* Of all the projects you worked on in the past, what is your most favorite one and why?
* Any interesting comments or code you've seen?
* What are your most and least favorite features in Scala 2 and 3?
* If you can remove any one thing from Scala, what would you choose?
* I just heard that Netflix is making `Cowboy Bebop` TV series which is a live-action adaptation of the anime, `Cowboy Bebop`. I feel like they are ruining one of my favorite animes just like what happened to `Ghost in the Shell`. What's your opinion... oh wait. I think you told me that you hadn't seen Cowboy Bebop. Please let me know if you watch it in the future. 🙂

Thanks for your awesome work on so many open-source projects!
